styles.css

```diff
@@ -1097,6 +1097,20 @@ svg {
     line-height: 24px;
   }
 
+  .desktop_about {
```

Comment:
- [ Required ] Good job fixing the `top` margin of this section of your portfolio but as suggested by a previous reviewer please consider also making sure the `bottom` margin is also consistent with the Figma template provided. As a developer, you'll be expected to implement designs the way they are on design templates, kindly fix this.

## Figma template
![Screenshot from 2023-03-30 21-23-28](https://user-images.githubusercontent.com/16392046/228931010-144fc7f6-337e-4976-a71b-ce9e43bb2a60.png)

## Your design
![Screenshot from 2023-03-30 21-23-58](https://user-images.githubusercontent.com/16392046/228931250-c94390db-f569-4a94-927c-eaa7640da89b.png)

Reply:
Hi @Felix45 Felix45, thank you for the advise. But on my screen this is what am seeing and its the reason why I considered the changes done and submitted.
<img width="1409" alt="image" src="https://user-images.githubusercontent.com/88074420/228938567-ed629221-cc51-48b4-8762-c44d2eb4aaee.png">

Reply:
Hi @BenjaminMatembe let me reach out to you on zoom.

Reply:
Hi, I joined a call with the student and helped him fix the problem, please review his project.

Reply:
@Felix45 I will just take a quick look and review